tools/validators/instance_validator/tests/fake_instances/GOOD/entity_interdependency_v1_alpha.yaml

```diff
@@ -65,29 +65,29 @@ PHYSICAL-ENTITY-GUID:
 PHYSICAL-ENTITY-2-GUID:
   type: HVAC/CHWS_WDT
   code: PHYSICAL-2-ENTITY
-  cloud_device_id: "foobar2"
+  cloud_device_id: "2619178366980755"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.